### PR TITLE
multiple pages: remove trailing whitespace

### PR DESCRIPTION
**Pull request by schneiderl**

### CODE REVIEW

File: pages.pt_BR/common/xkill.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # xkill
 
 > Termina o cliente associado a um elemento gráfico.
-> Utilizado para forçar a terminação de processos que não respondem ou não apresentam botão "fechar" 
+> Utilizado para forçar a terminação de processos que não respondem ou não apresentam botão "fechar"
```
Comment (zdroid) [inline pages.pt_BR/common/xkill.md]:
```suggestion
> Utilizado para forçar a terminação de processos que não respondem ou não apresentam botão "fechar".
```

**Comment by schneiderl** [comment]
There are also a lot of Chinese pages with trailing whitespaces. I didn't remove them because I'm not sure if this is a thing on the Chinese language. 
Anyone who speaks Chinese around here?

**Comment by mebeim** [comment]
@schneiderl those trailing whitespaces are 99.99% of the times mistakes. I've seen a lot of them in translated pages, they should be removed.

**Comment by schneiderl** [comment]
@mebeim done.

**Pull request merged**